app/views/users/show.html.slim

```diff
@@ -1,12 +1,29 @@
-.row
-  h2.show My profile
-  .small-5.small-offset-2.columns
+.row.main-content
+  div
     = image_tag @user.avatar.url(:medium)
-  .small-5.columns.profile
-    strong Name: 
-    = @user.first_name + ' ' + @user.last_name
-    br
-    strong E-mail: 
-    = @user.email
+  .profile
+    h2 #{@user.first_name + ' ' + @user.last_name}
+    - if @user.try(:speaker?)
+      .speaker-label Speaker
+    .info
+      - unless @user.title.nil?
+        p #{@user.title}
+      - unless @user.work_since.nil?
+        p Started working #{distance_of_time_in_words(@user.work_since, DateTime.now.utc, include_seconds: false)} ago
     br
+    div.container
+      h4 About
+      - unless @user.desc.empty?
+        p #{@user.desc}
+      - else
+        p No information yet
     = link_to 'Edit profile', edit_user_path(current_user)
+  div
+    .container.info
+      h3 Contacts
+      = image_tag "mail.png"
+      = @user.email
+      - unless @user.phone.nil?
+      br
+        = image_tag "phone.png"
```

Comment:
in such cases you can use some icon fonts such as https://design.google.com/icons/ or https://fortawesome.github.io/Font-Awesome/